community/sample-apps/whatsapp-clone-typescript-react/hasura/migrations/whatsapp-db.sql

```diff
@@ -0,0 +1,439 @@
+--
```

Comment:
This file does not conform to Hasura's migration naming scheme. Will not get applied with `hasura migrate apply` as mentioned in the readme.

Reply:
Renamed to 1_whatsappdb.sql